src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,511 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Created with Cambalache 0.10.2 -->
+<interface>
+  <!-- interface-name dialog-vkbasalt.ui -->
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="vkBasaltDialog" parent="AdwWindow">
+    <property name="default-width">550</property>
+    <property name="deletable">False</property>
+    <property name="modal">True</property>
+    <property name="title" translatable="yes">vkBasalt Settings</property>
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="AdwHeaderBar">
+            <child type="start">
+              <object class="GtkButton" id="btn_cancel">
+                <property name="label" translatable="yes">Cancel</property>
```

Comment:
It might be a good idea to allow mnemonics here, (you press `alt` and a underline appears bellow a letter of the button so pressing that letter activated the button). 

E.g.

```xml
                <property name="label" translatable="yes">_Cancel</property>
                <property name="use_underline">True</property>
```